public/_headers

```diff
@@ -0,0 +1,47 @@
+# Dialect Schemas
+/draft/:draft/*schema
+  Access-Control-Allow-Origin: *
+  Content-Type: application/schema+json
+  Cache-Control: public, max-age=31536000, immutable
+
+# Vocabulary Schemas
+/draft/:draft/meta/*
+  Access-Control-Allow-Origin: *
+  Content-Type: application/schema+json
+  Cache-Control: public, max-age=31536000, immutable
+
+# Links Schemas
+/draft/:draft/links
+  Access-Control-Allow-Origin: *
+  Content-Type: application/schema+json
+  Cache-Control: public, max-age=31536000, immutable
+
+# Output Schemas
+/draft/:draft/output/*schema
```

Comment:
```suggestion
/draft/:draft/output/*
```

I'll suggest this modification to cover "/draft/2019-09/output/verbose-example" and "/draft/2020-12/output/verbose-example"

Reply:
"verbose-example" doesn't belong because it's not a schema. It's example output for the verbose output format.